docs/zenodo.rst

```diff
@@ -321,6 +321,30 @@ users with access to your account can see their files.
     directory and upload the archive to Zenodo Sandbox, then download and unzip
     it the next time it is needed.
 
+    You will also need to update the rule for any scripts that use the output
```

Comment:
This is only necessary for the cases where you've already written a custom Snakemake rule for your script, right? The default rules will handle this properly as long as you update `showyourwork.yml`, right?

If so, it might be better to make that more explicit in this comment since I think it's a bit confusing right now…

Reply:
Ah yes I didn't think of the case where the inputs are not generated by your own rules. 

I did some more testing and found the syntax to be much more flexible than I had initially realised (I must have had other bugs during my initial tests). I will update this section to remove the rule output and show that you can point to the directory.